resources/WorkRecordResource.json

```
@@ -24,9 +22,13 @@
         {
             "$ref":"../enums/WorkItemPriorityType.json"
         },
-        "area": {
+        "workedArea": {
             "$ref": "../types/AreaMeasureType.json",
-            "description": "Area of the operation"
+            "description": "Area worked by the operation"
+        },        
+        "nonOverlapArea": {
```

Consider renaming this?
```suggestion
        "nonOverlapWorkedArea": {
```